vipps-recurring-api-faq.md

```diff
@@ -180,7 +180,7 @@ A charge will be displayed to the user 6 days before the charge is due to be pro
 The charge will then appear in the app.
 
 You can still retrieve all relevant charges through the API:
-[`GET:/recurring/v2/agreements/{agreementId}/charges`](https://vippsas.github.io/vipps-recurring-api/#/Charge%20Endpoints/listCharges).
+[`GET:/recurring/v3/agreements/{agreementId}/charges`](https://vippsas.github.io/vipps-recurring-api/#/Charge%20Endpoints/listCharges).
```

Comment:
Hi Amelie! I found that the links into these endpoints are broken because they don't include the 'v2' in the path. But, I'm not going to fix them, because it seems like you might be going to change them anyway. Should we add the version into all these urls (e.g.,`https://vippsas.github.io/vipps-recurring-api/#/Charge**%20v2%20**endpoints/ListCharges`)?

Reply:
But I guess I'll wait this one https://github.com/vippsas/vipps-recurring-api/pull/198 to be merge first